docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
+6. To export your private key for this account, click on the three dots next to the account name and select "Account Details".
     
-    ![](https://i.imgur.com/leSYt4z.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/83fffa23-4a3b-46f9-a492-9748bfd47cff)
 
-10. Switch to Sepolia Test Network.
-11. You can see that the same account can be used with different networks. But note that the ETH balance on different networks are different and are different tokens.
+   Select "Show Private Key".
     
-    ![](https://i.imgur.com/Y7YkuEe.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/3a513389-2df1-4e32-86da-a1794126cdac)
 
-12. Now to export your private key for the account, if needed, please click on the three dots beside the account.
+   Enter your Metamask password and click "Confirm"
     
-    ![](https://i.imgur.com/9BlBmhT.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/ffbac631-b933-4292-a2c6-dc445bff153c)
 
-13. Click on Account Details.
-14. Click on Export Private Key.
-    
-    ![](https://i.imgur.com/7g6SyuX.jpg)
+   You will be shown the private key. Copy it as needed.
 
-15. Enter your Metamask password when prompted.
-    
-    ![](https://i.imgur.com/VOojHY5.jpg)
-
-16. You will be shown the private key. Copy it as needed.
-
-Obtain Sepolia Ethers from faucet
----
-
-1. Ethers on Sepolia test networks can be obtained from different faucets.
-2. One of the faucets is as follows:
-    1. [https://sepoliafaucet.com//](https://sepoliafaucet.com/)
-    (NOTE: We have not tested the security of these faucets so please feel free to do your own research or obtain Sepolia ethers from other faucets if needed.)
-3. Please follow the instructions on the webpages of these faucets.
-4. A lot of faucets limit the Sepolia ETH to 0.05 ETH/day.
-5. To obtain more eth, you can do some POW mining. One of those POW faucet is:
-     [https://sepolia-faucet.pk910.de/](https://sepolia-faucet.pk910.de/)
-6. Enter your Eth account address, accept Captcha and start mining.
-    
-    ![](https://i.imgur.com/IvHNsei.jpg)
+## 2. Obtain Sepolia ETH from faucet
+
+Sepolia ETH can be obtained from different faucets.
+Three popular examples include:
+
+  1. [sepoliafaucet.com](https://sepoliafaucet.com/) (requires an Alchemy account)
+  2. [Infura Sepolia faucet](https://www.infura.io/faucet/sepolia) (requires an Infura account)
+  3. [Sepolia POW faucet](https://sepolia-faucet.pk910.de/)
+
+> _*NOTE:*_ This list is provided for convenience. We do not necessarily recommend or guarantee the security of any of these options.
 
-7. You can see the estimated Sepolia ETH mined per hour. Each session is restricted to a few hours.
+Many faucets limit the amount of Sepolia ETH you can obtain per day.
+To obtain more, a faucet based on PoW mining such as [sepolia-faucet.pk910.de](https://sepolia-faucet.pk910.de/) can be used:
```

Comment:
The PoW faucet also limit how much can be mined (per session and overall).

Also, do we need a separate instruction for the PoW faucet at all?

Reply:
Mmm, yeah I also considered just removing this section, but decided to just focus on getting what we have to be a bit more accurate. I've changed the phrasing here in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890. I guess it's convenient to have an example, although it's pretty straightforward.